composer.json

```diff
@@ -18,6 +18,7 @@
         "illuminate/support": "~5.3.0|~5.4.0",
         "intervention/image": "^2.3",
         "doctrine/dbal": "^2.5",
+        "caffeinated/themes": "^3.0",
```

Comment:
We can remove this from requirements and put it as a suggestion, and just note users that they need this when enabling the `use_theme` setting.